develop/reference/articles/01-intro/00-development-reference-intro.markdown

```diff
@@ -1,113 +1,61 @@
 # Development Reference [](id=development-reference)
 
 <p style="margin-left: 40px;">
-Here you'll find Liferay's reference documentation for Java, tag libraries,
+This page gives you access to reference documentation for Java, tag libraries,
 JavaScript and CSS, and deployment descriptors.
 </p>
 
 <p style="margin-left: 40px;">
 Liferay Portal 7 Community Edition's Portal Core reference documentation is
 available online  at <a href="https://docs.liferay.com/portal/7.0"
-target="_blank">https://docs.liferay.com/portal/<span
+target="_blank">https://docs.liferay.com/portal/7.0<span
 class="opens-new-window-accessible"> (Opens New Window)</span></a>
-and available for download as a ZIP file from Liferay's Downloads
-<a href="https://www.liferay.com/downloads" target="_blank">Liferay's Downloads<span
+and from the
+<a href="https://www.liferay.com/downloads" target="_blank">Downloads page<span
 class="opens-new-window-accessible"> (Opens New Window)</span></a>. 
 </p>
 
 <p style="margin-left: 40px;">
-Liferay Portal CE module reference documentation JAR files accompany each
-module's class artifacts in the <a
-href="https://repository.liferay.com/nexus/content/repositories/liferay-releases-ce/com/liferay/"
-target="_blank">Liferay Nexus repository<span
-class="opens-new-window-accessible"> (Opens New Window)</span></a>. The reference
-documentation includes Javadoc and tag library (taglib) docs.
+Liferay Portal CE module Javadoc and taglib doc JAR files are available in the
+<a href="https://repository.liferay.com/nexus/content/repositories/liferay-releases-ce/com/liferay/" target="_blank">
+Liferay CE artifact repository<span class="opens-new-window-accessible"> (Opens New Window)</span></a>.
```

Comment:
Same as for line 11, except that in this case this URL takes to an even more confusing place with tons of directories and subdirectories. Do we really need to provide this URL? If so, does it need to be up here where it's so visible?